R/setup.R

```diff
@@ -27,11 +23,7 @@
 #'
 #' @export
 use_travis_deploy <- function(path = usethis::proj_get(),
-                              user = github_user()$login,
-                              repo = github_info(
-                                path = path,
-                                remote = remote
-                              )$name,
+                              repo = get_repo_slug(remote),
```

Comment:
Do we need a `remote` argument?

Reply:
Imo yes because this enables the dynamic use of `use_travis_deploy()` in the first place.

See the underlying `github_info()`

https://github.com/ropenscilabs/travis/blob/e2a3498957aaf44ab60d69956294984e4d664b23/R/github-key.R#L110-L115

Reply:
Only saw the argument now.